docs/frameworks/typesafe-project.md

```diff
@@ -13,10 +13,9 @@ import TabItem from '@theme/TabItem';
   <title>Getting Started with Genezio | Genezio Documentation</title>
 </head>
 
+Welcome to the Genezio Typesafe "Getting Started" guide! In this tutorial, you will learn how to write a backend class, export its methods, call them from the frontend code, and then deploy both the frontend and the backend locally for testing and pushing live to a staging environment.
```

Comment:
Rename the first header `# Getting started` and the title to `# Typesafe Project` otherwise it will appear as `Getting started` in the sidebar.

![image](https://github.com/user-attachments/assets/23c215e4-e551-4dea-8000-a364fb9e01ec)